src/app/messages-page/messages-thread/messages-thread.component.html

```diff
@@ -8,10 +8,8 @@
     'message-thread__unread-light-blue':
       !isHovered &&
       globalVars.messageResponse &&
-      globalVars.messageResponse.TotalMessagesByContact[thread.PublicKeyBase58Check] !== undefined &&
-      globalVars.messageResponse.MessageReadStateByContact[thread.PublicKeyBase58Check] !== undefined &&
-      globalVars.messageResponse.TotalMessagesByContact[thread.PublicKeyBase58Check] !=
-        globalVars.messageResponse.MessageReadStateByContact[thread.PublicKeyBase58Check]
+      globalVars.messageResponse.UnreadStateByContact[thread.PublicKeyBase58Check] !== undefined &&
+      globalVars.messageResponse.UnreadStateByContact[thread.PublicKeyBase58Check] === true
```

Comment:
you can just check `globalVars.messageResponse.UnreadStateByContact[thread.PublicKeyBase58Check]`